08_files_lab/01.py

```diff
@@ -1,6 +1,5 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+import sys
+with open (sys.argv[1], "r") as fin:
+	with open (sys.argv[2], "w") as fout:
+		for line in fin:
+			fout.write(line)

```

Comment:
Hi All and Sorry,

That's actually a translation error on my part. Comments should say the other way around (so this is the correct solution). New fixed versions of the starter is committed.